### PR TITLE
Prettify queries using `sql-log-prettifier`

### PR DESCRIPTION
**Pull request by vladimyr**
This PR swaps default bunyan cli with custom [`bunyan-pretty-cli`](https://github.com/vladimyr/bunyan-pretty-cli) which enables prettifying custom keys. [`sql-log-prettifier`](https://npm.im/sql-log-prettifier) has been used for prettifying sequelize query log.

<a href="https://asciinema.org/a/pmnSDDJchLLyWXP3pccrtqGhl" target="_blank"></a>

### CODE REVIEW

File: bunyan-pretty.config.js [hunk 1]
```diff
@@ -0,0 +1,15 @@
...
+const { format } = require('util');
+const { prettify } = require('sql-log-prettifier');
+
+module.exports = {
+  level: 'debug',
+  customPrettifiers: {
+    query(input) {
+      const index = input.indexOf(': ') + 1;
+      const sql = input.substring(index);
+      return format('\n```sql\n%s\n```', prettify(sql));
```
Comment (zcuric) [inline bunyan-pretty.config.js:12]: `prettify` accepts `settings` as the second argument, have you considered changing some colors?

File: package.json [hunk 2]
```diff
@@ -141,6 +141,7 @@
     "@poi/plugin-eslint": "^12.0.0",
     "@types/poi": "^12.5.0",
     "babel-eslint": "^10.0.3",
+    "bunyan-pretty-cli": "github:vladimyr/bunyan-pretty-cli",
```
Comment (zcuric) [inline package.json]: So are we sticking with bunyan or are we we considering moving to something that is more maintained like `pino`?

File: bunyan-pretty.config.js (hunk 1 above)
Comment (vladimyr) [reply bunyan-pretty.config.js:12]: This is POC in its true meaning so likely some things should be changed 😀

File: package.json (hunk 2 above)
Comment (vladimyr) [reply package.json]: Well I made it to make sure we get pretty logs even something prevents us to go with pino. Using this is no brainer, switch to pino needs to be discussed more broadly but atm I'm all for it.

Comment (zcuric) [reply package.json]: 👍

**Comment by underscope** [comment]
@vladimyr Any chance this will land soon?

**Comment by zcuric** [comment]
FYI, i've implemented this on @ExtensionEngine/cepp

**Pull request merged**